mysql-test/suite/rocksdb/t/allow_no_primary_key.test

```diff
@@ -96,3 +96,31 @@ DELETE FROM t1 WHERE a = 35 AND b = 'foo';
 --sorted_result
 SELECT * FROM t1;
 DROP TABLE t1;
+
+--echo #
+--echo # Issue #834/MDEV-15304 ALTER TABLE table_with_hidden_pk causes Can't
+--echo #   write; duplicate key in table error and/or crash
+--echo #
+CREATE TABLE t1 (a INT, KEY(a)) ENGINE=RocksDB;
+INSERT INTO t1 VALUES (1),(1+1);
+create table t2 (a int);
```

Comment:
Do I understand correctly that t2 engine is not important as long as it's transactional?

Reply:
No, I think it needs to be MyRocks.
(t2 is present in the test to cause MyRocks to start a transaction. It is a MyRocks table in my setup. I didn't try other engines for it but I don't think using another engine will cause MyRocks transaction to be created)

I'll add ENGINE=RocksDB.